packages/terra-form-select/tests/jest/Menu.test.jsx

```diff
@@ -107,4 +110,49 @@ describe('Menu', () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should call focus on select component when select is closed via key event while not using safari browser', () => {
+    SharedUtil.isSafari.mockImplementation(() => false);
```

Comment:
We can use `mockReturnValueOnce` since we're only returning a value!
```suggestion
    SharedUtil.isSafari.mockReturnValueOnce(false);
```